_posts/2017-02-03-updates.markdown

```diff
@@ -0,0 +1,9 @@
+---
+layout: post
+title:  Terms of Reporting
+date:   2017-02-03 08:00:00
+categories: updates
+---
+Today we've updated to __v3.26.0__, which implements reports for all vocabulary terms in the MyReports tool. to find out just what reports are available, please visit [our reports grid](https://www.iliosproject.org/myreports.html).
+
+We've also added a Docker version of the Ilios deployment ([contact us](mailto:support@iliosproject.org?subject=docker) for more info), and pushed out a bunch of great updates to the system, and would love to hear back from you about what you think! Write us at [support@iliosproject.org](mailto:support@iliosproject.org?subject=feedback)!
```

Comment:
Remove `Write us` and have the last sentence just be something like `Tell us what you think at support@iliosproject.org!`